infra-parent/infra-v2/src/main/java/com/redhat/service/smartevents/infra/v2/api/models/dto/BridgeDTO.java

```diff
@@ -36,6 +37,9 @@ public class BridgeDTO {
     @JsonProperty("operation")
     private OperationType operationType;
 
+    @JsonProperty("deletionRequestedAt")
```

Comment:
Can we have `generation` too please?

You're yet to provide an alternative solution to the problem we described.

Reply:
done